clients/venice-push-job/src/main/java/com/linkedin/venice/hadoop/VenicePushJob.java

```diff
@@ -2428,7 +2430,10 @@ void pollStatusUntilComplete(
               .append(pushJobSetting.veniceControllerUrl)
               .append("\ncontroller response: ")
               .append(response);
-
+          if (overallStatus.isDVCIngestionError()) {
+            this.pushJobDetails.pushJobLatestCheckpoint =
+                PushJobCheckpoints.valueOf(overallStatus.toString()).getValue();
```

Comment:
Shall we have a try/catch here in case the DVC error is not recognized when we add more DaVinci error status, but H2V hasn't been updated yet?

Reply:
Thanks for catching this.
The code at line 2410 will fail first for the same reason. 
`      ExecutionStatus overallStatus = ExecutionStatus.valueOf(response.getStatus());`
Curious whether we should rely on deployment order here (VPJ driver before DVC client) or add some work around code. Work around only makes sense if we assume that the unknown status are DVC specific errors, if not it can be any random newly added status which can mean anything.

Reply:
Thinking further, added a function `getExecutionStatusFromControllerResponse` for getting `overallStatus` which throws a detailed exception in case of mismatches.